task_schemas.md

```diff
@@ -41,22 +42,23 @@ This is a simple container format with minimal nesting that supports a range of
 
 **Schema Notes**
 
-- `id` must be unique string within dataset and across parent category (e.g., entities, passages)
+- `id` these can be set in any fashion that makes every `id` field in a dataset unique.
```

Comment:
I'd mention the uniqueness across categories explicitly here

Reply:
thanks, I will make this more clear!